recipes/pandas/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version="0.18.1" %}
+
+package:
+    name: pandas
+    version: {{ version }}
+
+source:
+    fn: pandas-{{ version }}.tar.gz
+    url: https://github.com/pydata/pandas/archive/v{{ version }}.tar.gz
+    sha256: 1f3136f69a58d490ba9f9874e67601042a11138270515dedcf6979c003bba79b
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --root=/
```

Comment:
@msarahan and @jakirkham please do not change what you both are doing!

@jakirkham you reviews are very important for consistency and quality of the recipes.

@msarahan you presence here is wanted! But of course I know you don't have the time to keep fixing every little detail.

Here is a compromise: send a broken and/or inconsistent recipe and ping me :wink: I will gladly take over and do the boilerplate stuff.